ast/parser_test.go

```diff
@@ -2662,6 +2662,146 @@ else = {
 	`), curElse.Head.Value.Location)
 }
 
+func TestGetAnnotation(t *testing.T) {
+
+	tests := []struct {
+		note              string
+		testModule        string
+		expNumComments    int
+		expNumAnnotations int
+		expAnnotations    []*SchemaAnnotation
+		expError          string
+	}{
+		{
+			note: "Single valid annotation",
+			testModule: `
+			package opa.examples
+		
+			import data.servers
+			import data.networks
+			import data.ports
+		
+			#Schema annotation for this rule referencing three schemas
+			#@rulesSchema=data.servers:schemas.servers
```

Comment:
I agree with you completely on expecting someone to separate the schemas with comma and space. So, makes me a bit nervous to keep this for later on improvement. How about adding just this one line to remove all spaces in the comment line...?
```
text := strings.ReplaceAll(string(comment.Text), " ", "")
if strings.HasPrefix(text, schemaAnnot) {
```

Reply:
In this case, I wonder if the mini-language is simple enough so that a regular expression wouldn't be a bad choice. What do you think?

This also sheds light on some other problem, data references could be more complicated:
```
> data
{
  "foo": {
    "bar baz": 100
  }
}
> data.foo["bar baz"]
100
```
if we wanted to put a schema on that, replacing all strings won't do.

Reply:
Would it be possible to use YAML to define the annotations? This would help with #2180. Something like:

```rego
package x

import data.acl

# type: rule
# title: Enumerates users who can perform operation
# schemas:
#   input: schemas.foo.bar
#   data.acl: schemas.baz
whocan[user] {
        access = acl[user]
        access[_] == input.operation
}
```

For this PR we could start w/ something minimal like just the "schemas" key.

Reply:
@tsandall @srenatus Schema annotations in Metadata yaml support added now (7a9983b0039987b079f04e2253b0159130aa1d48). Please let us know what you think.